Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -529,16 +529,27 @@ class BocList
 
   private static FixValidationErrorOverflowSize(scrollableContainer: HTMLElement)
   {
-    // Sets the width of the validation error field to be as wide as the visible part of the bocList.
+    // Sets the width of the validation error field to be as wide as the visible part of the bocList
+    // minus how much space the td on the left requires to align with the error indicator column.
     // This enables sticky behaviour while still allowing for a scrollbar if required.
 
-    const width = LayoutUtility.GetWidth(scrollableContainer);
-
     const overflowContainer: Nullable<HTMLElement> = scrollableContainer.querySelector('div.bocListTableBlock tr.bocListValidationRow div');
-    if (overflowContainer)
+    if (!overflowContainer)
     {
-      overflowContainer.style.width = "calc(" + width.toString() + "px - var(--remotion-themed-scrollbar-width))";
+      return;
     }
+
+    const tdBeforeValidationErrorColumn: Nullable<HTMLElement> = scrollableContainer.querySelector('div.bocListTableBlock tr.bocListValidationRow td');
+
+    let widthBeforeValidationFailureColumn = 0;
+    if (tdBeforeValidationErrorColumn)
+        // +1 for rounding errors
+      widthBeforeValidationFailureColumn = LayoutUtility.GetWidth(tdBeforeValidationErrorColumn) + 1;
+
+    // -1 for rounding errors
+    const availableWidthInContainer = LayoutUtility.GetWidth(scrollableContainer) - 1;
+
+    overflowContainer.style.width = "calc(" + (availableWidthInContainer - widthBeforeValidationFailureColumn).toString() + "px - var(--remotion-themed-scrollbar-width))";
```

Comment:
We missed this bug in the earlier PR: must not use css-variables like this in Javascript. They are theme-specific, not a contract.